### PR TITLE
Change `direct` to `unsanitize` to indicate the type of clipboard content

### PR DESCRIPTION
**Pull request by snianu**
Addressing security feedback.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -54,18 +55,18 @@ const image = await clipboardItem.getType('image/png');
 
 ## Pickling Write
 Pickled formats will be written when the site tries to use a format that:
-1. Is included in a `{direct: ['format1', 'format2']}` list of formats for the `ClipboardItem`’s `ClipboardItemOption`. 
+1. Is included in a `{unsanitize: ['format1', 'format2']}` list of formats for the `ClipboardItem`’s `ClipboardItemOption`.
```
Comment (dway123) [inline explainer.md]: Would `unsanitized` be better here? (`unsanitize` may be a misspelling)

File: explainer.md [hunk 2]
```diff
@@ -54,18 +55,18 @@ const image = await clipboardItem.getType('image/png');
...
+For example, writing the format `"text/plain"`, which is recognized by the Clipboard API as a sanitized format, will write this format to the clipboard as usual. On the other hand, writing a format `"text/custom-format"`, while also providing a `{unsanitize: ['text/custom-format']}` list will result in a pickled format corresponding to `"text/custom-format"` to be written to the clipboard. And writing “text/plain” with a `{unsanitize: ['text/plain']}` list will write both a sanitized and pickled payload. This API change allows a potential related pickling and sanitized format to coexist on the clipboard at the same time, which allows sites to interoperate with the more widely used sanitized format, while also getting the increased fidelity of the unmodified pickled format.
```
Comment (dway123) [inline explainer.md]: optional nit: For a tiny bit more info, we could say "On the other hand, writing a format text/custom-format, which is not recognized by the Clipboard API as a sanitized format, will result in a pickled format [...]". Not sure if that's too wordy though

Comment (snianu) [reply explainer.md]: That sounds good. Should we also include the `{unsanitized: ['text/custom-format']}` part as that is required for pickling write (even for custom formats)?

File: explainer.md [hunk 3]
```diff
@@ -122,32 +123,49 @@ On Linux, ChromeOS, and Android, MIME types are often used (though Linux and Chr
...
+
+Websites or native apps need to explicitly opt-in to consume these formats which will mitigate the concerns about remote code execution in legacy apps. Popular standardized data types (html, text, image etc) are available across all platforms and some types have sanitizers(html format) to strip out `<script>` and `comment` tags and decoders(for image formats), but for custom formats the content is unsanitized and could open up (by-design) a whole new world of attacks related to data types. This feature adds a [user gesture requirement](https://github.com/dway123/clipboard-pickling/blob/main/explainer.md#user-gesture-requirement) on top of [existing](https://github.com/dway123/clipboard-pickling/blob/main/explainer.md#permissions) async clipboard API security measures to mitigate security and privacy concerns.
```
Comment (dway123) [inline explainer.md]: nit: space between "sanitizers" and "(html format)"

File: explainer.md (hunk 2 above)
Comment (dway123) [reply explainer.md]: What do you mean by "the `{unsanitized: ['text/custom-format']}` part"?

**Review by dway123 — APPROVED** [review]
LGTM after comments are addressed. Thank you!

File: explainer.md (hunk 2 above)
Comment (snianu) [reply explainer.md]: So if we just provide `"text/custom-format"` in the `ClipboardItem` without `{unsanitize: ['text/custom-format']}`, then it shouldn't write any formats because `unsanitize` option is required for any format write in unsanitized form right?

File: explainer.md (hunk 1 above)
Comment (snianu) [reply explainer.md]: Yes, sorry it was a typo.

File: explainer.md (hunk 2 above)
Comment (dway123) [reply explainer.md]: Ah yes, providing an unsanitized format without the unsanitized flag shouldn't write any formats. Thanks for clarifying

Comment (snianu) [reply explainer.md]: Changed it to: On the other hand, writing a format `"text/custom-format"` with a `{unsanitize: ['text/custom-format']}` list, which is not recognized by the Clipboard API as a sanitized format, will result in a pickled format corresponding to `"text/custom-format"` to be written to the clipboard. And writing “text/plain” with a `{unsanitized: ['text/plain']}` list will write both a sanitized and pickled payload.
